components/actionbutton/index.css

```diff
@@ -22,6 +23,9 @@ governing permissions and limitations under the License.
   /* todo: remove hardcoded values when we update DNA */
```

Comment:
Has this been addressed in this work?

Reply:
Nah, looks like they're still wrong in DNA or need to be overridden due to the border model.